extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -278,19 +278,30 @@ export class AmpA4A extends AMP.BaseElement {
           return Promise.reject(cancellation());
         }
         if (fetchResponse && fetchResponse.arrayBuffer) {
-          return fetchResponse.arrayBuffer().then(
-            bytes => {
-              if (promiseId != this.promiseId_) {
-                return Promise.reject(cancellation());
-              }
-              return this.validateAdResponse_(fetchResponse, bytes)
-                .then(valid => {
+          return fetchResponse.arrayBuffer().then(bytes => {
+            if (promiseId != this.promiseId_) {
+              return Promise.reject(cancellation());
```

Comment:
Nifty!  And that will become the arg to the `.catch()` branch of the enclosing Promise, I assume?

This is a good cleanup, but, if you don't mind, I'd prefer to do it in a separate PR, because we should do it all through `#onLayoutMeasure`.  And that makes the purpose of this PR less clear.

If you feel strongly, though, I can do it here.